### PR TITLE
[VET-5320] add hubspot events

### PR DESCRIPTION
**Pull request by mysticfalconvt**
[VET-5320]
Adding trackEvent from cloud-login to portal SDK

Types created by changing the codegen.yml and using updated local build of cloud-login.

### CODE REVIEW

File: src/getPortalSdk.ts [hunk 1]
```diff
@@ -2,6 +2,34 @@ import { GraphQLClient } from 'graphql-request';
 import { getConnectionCookie } from './cookies';
 import { getSdk, ShareInput } from './sdk';
 
+export enum trackingEventList {
```
Comment (anneeb) [inline src/getPortalSdk.ts:5]: nit: title case for enum `TrackingEventList`

File: src/getPortalSdk.ts [hunk 2]
```diff
@@ -2,6 +2,34 @@ import { GraphQLClient } from 'graphql-request';
...
+  STUDIO_CREATE_USER = 'Studio create user',
+  STUDIO_CREATE_VIRTUAL_GRAPH = 'Studio create virtual graph',
+  STUDIO_EDIT_DATA_SOURCE = 'Studio edit data source',
+  STUDIO_EDIT_MODEL = 'Studio edit model',
+  STUDIO_EDIT_VIRTUAL_GRAPH = 'Studio edit virtual graph',
+  STUDIO_SPARQL_QUERY_EXECUTION = 'Studio SPARQL query execution',
+  VERIFICATION_MODAL_SEEN = 'Verification modal seen',
+}
+
+export enum clientTypeList {
```
Comment (anneeb) [inline src/getPortalSdk.ts:29]: nit: title case for enum `ClientTypeList`

File: src/getPortalSdk.ts [hunk 3]
```diff
@@ -22,6 +50,13 @@ export const getPortalSdk = () => {
       const result = await sdk.addShare({ input });
       return result.addShare || null;
     },
+    TrackEvent: async (input: {
```
Comment (anneeb) [inline src/getPortalSdk.ts:53]: nit: lowercase for function methods `trackEvent`

Comment (mysticfalconvt) [reply src/getPortalSdk.ts]: cb1fd351549d60675574453f5bc999c7ea07ac42

File: src/getPortalSdk.ts (hunk 1 above)
Comment (mysticfalconvt) [reply src/getPortalSdk.ts]: cb1fd351549d60675574453f5bc999c7ea07ac42

File: src/getPortalSdk.ts (hunk 2 above)
Comment (mysticfalconvt) [reply src/getPortalSdk.ts]: cb1fd351549d60675574453f5bc999c7ea07ac42

**Review by anneeb — APPROVED** [review]

**Review by SpiralP — APPROVED** [review]

**Pull request merged**